app/section/helpers/transformSectionData.js

```diff
@@ -10,10 +10,15 @@ const arrayTransformation = (questionValue) => {
   return [questionValue];
 };
 
+const radioButtonTransformation = questionValue => (questionValue || null);
```

Comment:
I would name this function to something like 'emptyValueTransformer' or something along them lines. We could use this for something else?

Reply:
it was stringTranformation until API got involved ... 😄